tests/test_vqe.py

```diff
@@ -412,7 +413,7 @@ def test_optimize(self, interface, tf_support, torch_support):
         assert exec_opt == 5  # Number of groups in the Hamiltonian
         assert exec_no_opt == 15
 
-        assert np.allclose(c1, c2)
+        assert np.allclose(c1, c2, atol=1e-1)
```

Comment:
This tolerance seems quite large, is that expected?

Reply:
Yes, with a standard number of shots like 8000, I think it is normal. Otherwise we can increase the shots and the tolerance but it takes more time.